configs/phalcon.json

```diff
@@ -20,12 +11,21 @@
       "global": true,
       "default_value": "Documentation"
     },
-    "lvl1": {
-      "selector": "???"
-    },
-    "lvl2": ".DocsContainer article h1",
-    "lvl3": ".DocsContainer article h2",
-    "lvl4": ".DocsContainer article h3",
+    "lvl1": ".DocsContainer article h1",
```

Comment:
This is not correctly grabbing the title. For example on https://phalc.one/docs/3.4/config, `lvl1` should be "Config" (as selected in the sidebar), not `Reading Configurations`.

Reply:
Yep I thought about that. We will have duplicate if we do that. Let's do that and we will iterate if needed.

Reply:
What kind of duplicate?

Reply:
Duplicate from the website itself. Title remind the lvl0 in their text